kw_webapp/models.py

```diff
@@ -271,15 +273,19 @@ class UserSpecific(models.Model):
     class Meta:
         unique_together = ("vocabulary", "user")
 
-    def answered_correctly(self, first_try=True):
+    def answered_correctly(self, first_try, can_burn):
         # This is a check to see if it is a "lesson" object.
         if self.streak == 0:
             self.streak += 1
         elif first_try:
             self.correct += 1
             self.streak += 1
             if self.streak >= constants.WANIKANI_SRS_LEVELS[WkSrsLevel.BURNED.name][0]:
-                self.burned = True
+                # If can burn, do so. Otherwise, keep review at enlightened.
+                if can_burn:
+                    self.burned = True
+                else:
+                    self.streak = constants.WANIKANI_SRS_LEVELS[WkSrsLevel.ENLIGHTENED.name][0]
```

Comment:
Would have expected `KANIWANI_SRS_LEVELS` and `KwSrsLevel` to be used in the function instead, but I guess it doesn't matter too much.

Reply:
Youre right. If they are the same value, feel free to swap em

Reply:
Done, fixed a couple of other places where it was imported and not used.